release_notes/48-release-notes.adoc

```diff
@@ -457,4 +457,27 @@ This release also addresses the following security vulnerabilities:
 
 * Vulnerability in the form-data JavaScript library (link:https://access.redhat.com/security/cve/cve-2025-7783)[CVE-2025-7783])
 
+
+[id="about-this-release-485_{context}"]
+== About release 4.8.5
+
+*Release date* {ga-date-485}
+
+This release provides the following bug fixes:
+
+//ROX-30462
+* Before this update, Central reduced its logging from `ERROR` to `DEBUG` when semaphore acquisition failures occurred during shutdown. As a consequence, the log files were filled with excessive, misleading error messages when multiple scans were queued, making it difficult to identify genuine shutdown problems.
++
+With this update, the log level for semaphore acquisition failures is reduced from `ERROR` to `DEBUG`. As a result, it is easier to identify actual shutdown issues without the distraction of log spam.
+
+//ROX-30867
+* Before this update, Central processed large batches of process indicators in a single database transaction while holding a lock. As a consequence, you experienced mutex timeouts, and Central and Central DB were strained, leading to lock contention and prolonged transaction durations.
++
+With this update, the process indicator processing logic is optimized to minimize lock contention and transaction duration. As a result, mutex timeouts are prevented, and strain on Central and Central DB during high-volume indicator processing is reduced.
```

Comment:
This seems really wordy. I might consolidate:

Before this update, Central processed large batches of process indicators in a single database transaction while holding a lock. With this update, the process indicator processing logic is optimized, resulting in reduced strain on Central and Central DB during high-volume indicator processing.